cypress/support/pageObjects/performancePageLocators.js

```diff
@@ -0,0 +1,335 @@
+class performancePageLocators {
```

Comment:
```suggestion
class PerformancePage {
```
Since we're following the Page Object Model, would it make sense to define this as class `PerformancePage`